story1.html

```diff
@@ -1,18 +1,17 @@
 <!DOCTYPE html>
 <html>
-<head style= "background-color: #e5ba57">
+<head class= "story1Head">
 	<meta charset="utf-8">
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 	<title></title>
-	<link rel="stylesheet" href="">
-	<link rel="stylesheet" href="style.css">
+	<link rel="stylesheet" type="text/css" href="/home/mnixon/Sites/htmlPractice/styles.css">
```

Comment:
Use a relative link here, so that if the projects location changes, your HTML document can still figure out where the CSS file is located.